pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -63,6 +63,12 @@ self: super: {
   # works fine there.
   fakedata = dontCheck super.fakedata;
 
+  # The latest release on hackage has an upper bound on containers which
+  # breaks the build, though it works with the version of containers present
+  # and the upper bound doesn't exist in code anymore:
+  # > https://github.com/roelvandijk/numerals
+  numerals = doJailbreak (dontCheck super.numerals);
```

Comment:
Why are tests disabled?

Reply:
That was from before so I didn't touch it, not sure if it'd build fine without it now but can try in a few.

Reply:
Ah right, never mind.